src/components/CounterBadge/CounterBadge.st.css

```diff
@@ -0,0 +1,69 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, applyOpacity, fallback;
+}
+
+/*Overrides*/
+:vars {
+    fixedFont45: HelveticaNeueW01-45Ligh, HelveticaNeueW02-45Ligh, HelveticaNeueW10-45Ligh, Helvetica Neue, Helvetica, Arial, メイリオ, meiryo, ヒラギノ角ゴ pro w3, hiragino kaku gothic pro, sans-serif;
+}
```

Comment:
You should import it from the fonts.st.css file.

:import {
    -st-from: "../../common/fonts.st.css";
    -st-named: fixedFont45;
}